deno/gateway/v10.ts

```diff
@@ -1635,22 +1631,47 @@ export interface GatewayThreadCreateDispatchData extends APIThreadChannel {
 /**
  * https://discord.com/developers/docs/topics/gateway-events#thread-update
  */
-export type GatewayThreadUpdateDispatch = GatewayChannelModifyDispatch;
+export type GatewayThreadUpdateDispatch = DataPayload<
+	GatewayDispatchEvents.ThreadUpdate,
+	GatewayThreadUpdateDispatchData
+>;
 
 /**
  * https://discord.com/developers/docs/topics/gateway-events#thread-update
  */
-export type GatewayThreadUpdateDispatchData = GatewayChannelModifyDispatchData;
+export type GatewayThreadUpdateDispatchData = APIThreadChannel;
 
 /**
  * https://discord.com/developers/docs/topics/gateway-events#thread-delete
  */
-export type GatewayThreadDeleteDispatch = GatewayChannelModifyDispatch;
+export type GatewayThreadDeleteDispatch = DataPayload<
+	GatewayDispatchEvents.ThreadDelete,
+	GatewayThreadDeleteDispatchData
+>;
 
 /**
  * https://discord.com/developers/docs/topics/gateway-events#thread-delete
  */
-export type GatewayThreadDeleteDispatchData = GatewayChannelModifyDispatchData;
+export interface GatewayThreadDeleteDispatchData {
```

Comment:
This seems to be an extension of APIPartialChannel, let's reuse that please

Reply:
`APIPartialChannel` possibly has a name, whereas this event never does. Would it still be worth it to reuse that type, even though it's only for 2 fields (`id` and `type`)?

Reply:
oh. 😢 ok nvm me